### DISCUSSION TITLE
How to make Whisper recognise more words

### DISCUSSION THREAD

**Discussion by Abdurrafey-Siddiqui** [openai/whisper#288] [Q&A]
How do I manually add words to Whisper so that when it scans an audio file it can recognise the word and not mispell it?

**Comment by kundanashish** [comment]
> How do I manually add words to Whisper so that when it scans an audio file it can recognise the word and not mispell it?

Hi
Did you found any way to do it?

**Comment by glangford** [comment]
I have used `--initial_prompt` for misspelled non-English words successfully, have you tried that?

**Reply by dgoryeo** [reply]
I didn't know that would be an option. How is that done? Any examples? Thx

**Comment by glangford** [comment]
From the command line, you can add one or more words (delimited by spaces). 
`--initial_prompt "misspell"`

To do the same from Python, see the code and discussion in https://github.com/openai/whisper/discussions/355

**Reply by dgoryeo** [reply]
Thanks!

**Comment by giacomomasseron** [comment]
> From the command line, you can add one or more words (delimited by spaces). `--initial_prompt "misspell"`
> 
> To do the same from Python, see the code and discussion in #355

Sorry if I write wrong, but I am approaching whisper for the first time:

`result = model.transcribe("audio.mp3", initial_prompt='newword' )`

You use this code, and in the "audio.mp3" file there is a voice that says "newword", training the AI.
Am I right?
